src/dags/bouwstroompunten.py

```diff
@@ -102,15 +100,22 @@ def get_token() -> Union[str, Any]:
     # 2. Create temp directory to store files
     mkdir = mk_dir(tmp_dir, clean_if_exists=False)
 
+    # GET TOKEN
+    get_token = PythonOperator(
```

Comment:
Now that `get_token` (important: without the `_` prefix) is a `@task` we don't need the `PythonOperator` anymore. This is the Airflow Flow 2.0 API. Much more convenient than the old one. Whatever a `@task` returns is stored in XCOM under `xcom_key` with the function name as the value: `"get_token"` in this case. That's the reason I stripped the `_` prefix. After all `HTTPFetchOperator` expect the token to be retrieveable under the `xcom_key` `"get_token"`
```python
    get_jwt_token = get_token()
```